### PR TITLE
Correct uv_setup_args usage

### PR DESCRIPTION
**Pull request by cmfcmf**

### CODE REVIEW

File: src/node.cc [hunk 1]
```diff
@@ -4703,26 +4701,26 @@ int _StartEnv(int argc,
...
   PlatformInit();
   node::performance::performance_node_start = PERFORMANCE_NOW();
 
   // Hack around with the argv pointer. Used for process.title = "blah --args".
   // argv won't be modified
-  uv_setup_args(argc, const_cast<char**>(argv));
+  argv = uv_setup_args(argc, argv);
 
-  // This needs to run *before* V8::Initialize().
-  // Init() puts the v8 specific cmd args in exec_argc and exec_argv.
```
Comment (cmfcmf) [inline src/node.cc:4722]: This restores the original comment.

File: src/node.cc [hunk 2]
```diff
@@ -4703,26 +4701,26 @@ int _StartEnv(int argc,
...
+  // This needs to run *before* V8::Initialize().  The const_cast is not
+  // optional, in case you're wondering.
```
Comment (cmfcmf) [inline src/node.cc:4720]: Here it is :-D

**Review by luminosuslight — CHANGES_REQUESTED** [review]
I think we should keep the const-correctness where possible. I created an alternative PR #86 for this.

**Comment by cmfcmf** [comment]
Replaced by #86.

**Pull request closed without merging**